tgui/packages/tgui/interfaces/Changelog.jsx

```diff
@@ -183,31 +183,30 @@ export class Changelog extends Component {
 
     const header = (
       <Section>
-        <h1>Daedalus Dock</h1>
+        <h1>Dionysus</h1>
         <p>
           <b>Brought to you by: </b>
-          Kapu1178, Francinum, Spyroshark, Cenrus, FunnyLegJiggle, Zonespace,
```

Comment:
I don't mind keeping this here as acreditation. Kapu and Co. did quite alot to make Daedalus what it is; their names deserve to still be credited. We can probably split this into two fields; one for crediting them and one for the people who run the server; as that list is bound to grow with time. @ZephyrTFA